packages/curve-ui-kit/src/widgets/Disclaimer/Disclaimer.tsx

```diff
@@ -0,0 +1,73 @@
+import { useState } from 'react'
+import { t } from '@lingui/macro'
+
+import Stack from '@mui/material/Stack'
+
+import { TabsSwitcher } from 'curve-ui-kit/src/shared/ui/TabsSwitcher'
+import { SizesAndSpaces } from 'curve-ui-kit/src/themes/design/1_sizes_spaces'
+
+import { LastUpdated } from './LastUpdated'
+import { Footer } from './Footer'
+import { TabPanel } from './TabPanel'
+import { Dex } from './Tabs/Dex'
+import { LlamaLend } from './Tabs/LlamaLend'
+import { CrvUsd } from './Tabs/CrvUsd'
+import { SCrvUsd } from './Tabs/SCrvUsd'
+
+const { MaxWidth } = SizesAndSpaces
+
+const DEFAULT_TAB_INDEX = 0
+const tabs = [
+  { id: 'dex', label: t`Dex` },
+  { id: 'lend', label: t`LlamaLend` },
+  { id: 'crvusd', label: t`crvUSD` },
+  { id: 'scrvusd', label: t`Savings crvUSD` },
+].map((tab, value) => ({
+  ...tab,
+  value,
```

Comment:
this is confusing, why do we need a value here? `index` would be clearer
we could avoid the indexes completely and just use the ID
```
const DEFAULT_TAB = 'dex'
const tabs = {
  dex: t`Dex`,
  ......
}

Reply:
I had it as `index` before but `TabSwitcher` really wants a `value` prob. So I could use the ids, but I'd still have to rename the property from `id` to `value` as otherwise the compiler will complain

Reply:
How sbout `Object.entries(tabs).map(([value, label] => ....)`